ibm_db2/tests/test_unit.py

```diff
@@ -44,3 +48,26 @@ def mock_exception(*args, **kwargs):
             ibmdb2.check(instance)
         # new connection made
         assert ibmdb2._conn != conn1
+
+
+def test_metadata(aggregator, instance, datadog_agent):
+    check = IbmDb2Check('ibm_db2', {}, [instance])
+    check.check_id = CHECK_ID
+
+    check.check(instance)
+
+    # The version string differs from the image tag version
+    raw_version = "DB2 v11.1.2.2"
```

Comment:
Note to reviewer: the version tag used in the docker image is `11.1` where as the actual version string is `DB2 v11.1.2.2`. Open to suggestions to workaround when we add other versions. Our test environment currently only supports 11.1.